reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -638,7 +639,7 @@ protected void onOutboundComplete() {
 			f = channel().writeAndFlush(newFullBodyMessage(EMPTY_BUFFER));
 		}
 		else if (markSentBody()) {
-			LastHttpContent lastHttpContent = LastHttpContent.EMPTY_LAST_CONTENT;
+			LastHttpContent<?> lastHttpContent = new EmptyLastHttpContent(channel().bufferAllocator());
```

Comment:
same question: can we cache the EmptyLastHttpContent in a static variable and reuse it ?

Reply:
EmptyLastHttpContent is Resource, once you close it you cannot reuse it